cloud/pkg/devicecontroller/controller/downstream.go

```diff
@@ -40,6 +40,7 @@ import (
 	"github.com/kubeedge/kubeedge/cloud/pkg/devicecontroller/constants"
 	"github.com/kubeedge/kubeedge/cloud/pkg/devicecontroller/manager"
 	"github.com/kubeedge/kubeedge/cloud/pkg/devicecontroller/types"
+	"github.com/kubeedge/kubeedge/edge/pkg/devicetwin/dttype"
```

Comment:
I think it is strange that the cloudcore code import edgecore code, I think we can put the common type in the pkg/ directory

Reply:
I have moved the edge/pkg/devicetwin/dttype to pkg/common/dttype